Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandConfigurationView.swift

```diff
@@ -145,9 +145,7 @@ struct OnDemandConfigurationView: View {
             .onLayerViewStateChanged { _, _ in
                 mapIsReady = true
             }
-            // Prevent view from dragging when panning on map view.
-            .highPriorityGesture(DragGesture())
-            .interactiveDismissDisabled()
+            .preventMapInteractionFromMovingSheet()
```

Comment:
```suggestion
            .preventMapInteractionFromMovingSheet()
            .defersSystemGestures(on: .vertical)
```
Fixes drag gesture bug on the bottom of the sheet.